admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,251 @@
+[[admin-guide-cluster-capacity]]
+= Analyzing Cluster Capacity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can use the cluster capacity tool to view the
+number of pods that can be scheduled in order to increase the current resources
+before they become exhausted, and to ensure any future pods can be scheduled.
+This capacity comes from an individual node host in a cluster, and includes CPU,
+memory, disk space, and others.
+
+The cluster capacity tool simulates a sequence of scheduling decisions to
+determine how many instances of an input pod can be scheduled on the cluster
+before it is exhausted of resources in order to provide a more accurate
+estimation.
+
+[NOTE]
+====
+The remaining allocatable capacity is a rough estimation, because it does not
+count all of the resources being distributed among nodes. It analyzes only the
+remaining resources and estimates the available capacity that is still
+consumable in terms of a number of instances of a pod with given requirements
+that can be scheduled in a cluster.
+
+Also, pods might only have scheduling support on particular sets of nodes based
+on its selection and affinity criteria. As a result, the estimation of which
+remaining pods a cluster can schedule can be difficult.
+====
+
+The cluster capacity analysis tool can be run either as a stand-alone utility
```

Comment:
Can we change this to: 
`You can run the cluster capacity analysis tool as a stand-alone utility
from the command line, or as a job in a pod inside an {product-title} cluster. Running it as job inside of a pod enables you to run it multiple times without intervention.`

Also, link ` job inside of a pod ` to the `admin-guide-running-cluster-capacity-inside-pod` section